patterns/multisig/MultisigRune.ts

```diff
@@ -21,7 +22,7 @@ export interface Multisig {
 }
 
 // TODO: swap out `Chain` constraints upon subset gen issue resolution... same for other patterns
-export class MultisigRune<out C extends Chain, out U> extends Rune<Multisig, U> {
+export class MultisigRune<out C extends PolkadotSignatureChain, out U> extends Rune<Multisig, U> {
```

Comment:
This is a temporary suppression of the resulting type error (in lieu of proper constraints).